quiche/src/lib.rs

```diff
@@ -1339,6 +1347,25 @@ impl Config {
     pub fn set_disable_dcid_reuse(&mut self, v: bool) {
         self.disable_dcid_reuse = v;
     }
+
+    /// Set the server's preferred address.
```

Comment:
Let's add some further description of how we expect callers to use this function, with a link to the appropriate RFC section. Right now, it's too easy to do the wrong thing (v4 and v6 None, or empty CID) and get an error result that doesn't explain why.

Reply:
Good call! Added 👍